### ISSUE TITLE
[Accessibility] Zoom and scaling disabled for mobile on Core Profiler / Setup Wizard (WCAG 2.2 SC 1.4.4)

### ISSUE THREAD

**Issue by robcmcdowall** [woocommerce/woocommerce#67834]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The `<meta name="viewport">` element prevents users from zooming and scaling text because the `maximum-scale` value is set to `1.0`:

```html
<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
```

While some browser settings can override this restriction, iOS 10 and later ignores it by default. 

This issue is present on all of the **Core Profiler / Setup Wizard** onboarding screens. 

This accessibility issue impacts:

- WCAG 2.2 SC 1.4.4 (Resize Text).

### Expected behavior

Users should be able to scale up to at least 200% (`2.0`).

### Actual behavior

Zooming is disabled, preventing users from pinching to zoom on mobile devices.

## User impact

Because this issue occurs within the **Core Profiler / Setup Wizard**, it affects one of the first interactive experiences many users have with WooCommerce.

Users with partial site or low vision often choose to enlarge the fonts on their browser to make text easier to read and interact with. The broken zoom interaction introduces unnecessary friction during onboarding and makes the initial setup experience more difficult to complete on a mobile device.

## Proposed fix:

Remove the `maximum-scale` parameter from the `<meta name="viewport">` element:

```diff
- <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

### Open the Core Profiler / Setup Wizard

If the **Core Profiler / Setup Wizard** is not automatically displayed after installing WooCommerce, it can be reopened using the following steps:

1. Using an Android mobile device that does not have "Force Zoom" or a similar accessibility override enabled, log in to the **WordPress Admin** dashboard.
2. Tap the hamburger menu in the top-left corner.
3. Navigate to a WooCommerce page that displays the **Help** button (a "**?**" in a small circle) in the top-right corner, for example **WooCommerce** → **Home**, **Orders** or **Reports**.
4. Tap the **Help** button.
5. In the **Help** panel, select **Setup wizard** from the menu on the left.
6. In the **WooCommerce Onboarding** tab, tap the **Setup wizard** button in the **Profile Setup Wizard** section.

### Reproduce the issue

1. On the **Welcome to Woo** page, try to zoom using a pinch gesture.
2. Observe that zooming is disabled.
3. On the **Welcome to Woo** page, tap the **Set up my store** button.
4. On the "Which one of these best describes you?" page, try to zoom using a pinch gesture.
5. Observe that zooming is disabled.
6. Tap the **Continue** button.
7. On the "Tell us a bit about you?" page, try to zoom using a pinch gesture.
8. Observe that zooming is disabled.
9. Tap the **Continue** button.
10. On the "Get a boost with our free features" page, try to zoom using a pinch gesture.
11. Observe that zooming is disabled.
12. Tap the **Continue** button.
13. On the "Turning on the lights" page, try to zoom using a pinch gesture.

### WordPress Environment

<details><summary>System Status Report</summary>

```
### WordPress Environment ###

WordPress address (URL): [Redacted]
Site address (URL): [Redacted]
WC Version: 11.0.1
Action Scheduler Version: ✔ 4.0.0
Log Directory Writable: ✔
WP Version: 7.0.4
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
WP Environment Type: local
Language: en_GB
External object cache: –

### Server Environment ###

Server Info: nginx/1.26.1
Server Architecture: Windows NT 10.0 AMD64
PHP Version: 8.4.4
PHP Post Max Size: 1,000 MB
PHP Time Limit: 1200
PHP Max Input Vars: 4000
cURL Version: 8.10.1
OpenSSL/3.0.15

SUHOSIN Installed: –
MySQL Version: 8.0.16
Max Upload Size: 300 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

[REDACTED]

### Post Type Counts ###

attachment: 23
page: 8
post: 2
product: 18
product_variation: 7
revision: 10
shop_coupon: 1
shop_order_placehold: 3
wp_global_styles: 1
wp_navigation: 1

### Security ###

Secure connection (HTTPS): ❌
					Your store is not using HTTPS. Learn more about HTTPS and SSL Certificates.
Hide errors from visitors: ✔

### Active Plugins (1) ###

WooCommerce: by Automattic – 11.0.1

### Inactive Plugins (2) ###

Jetpack: by Automattic – 16.1.1
WooCommerce Stripe Gateway: by Stripe – 10.8.5

### Settings ###

Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: ✔
HPOS feature enabled: ✔
Order datastore: Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore
HPOS data sync enabled: –
Enabled Features: analytics
marketplace
order_attribution
site_visibility_badge
remote_logging
email_improvements
blueprint
point_of_sale
push_notifications
custom_order_tables


### Logging ###

Enabled: ✔
Handler: Automattic\WooCommerce\Internal\Admin\Logging\LogHandlerFileV2
Retention period: 30 days
Level threshold: –
Log directory size: 663 B

### WC Pages ###

Shop base: #7 - /shop/
Basket: #8 - /cart/ -  Contains the woocommerce/classic-shortcode block
Checkout: #9 - /checkout/ -  Contains the woocommerce/checkout block
My account: #10 - /my-account/ -  Contains the [woocommerce_my_account] shortcode
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 4.6.2
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build
personally we recommend using a child theme. See: How to create a child theme

Theme type: Classic theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
analytics-scheduled-import
experimental-iapi-mini-cart
coupons
core-profiler
customize-store
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
onboarding
onboarding-tasks
pattern-toolkit-full-composability
product-custom-fields
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
printful
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page
launch-your-store

Disabled Features: product-data-views
experimental-blocks
minified-js
settings-ui
rest-api-v4
order-detail-redesign
product-variations-classic-redesign

Daily Cron: ❌ Not scheduled
Options: ✔
Notes: 73
Onboarding: completed

### Action Scheduler ###

Complete: 17
Oldest: 2026-08-16 13:11:30 +0100
Newest: 2026-08-16 13:35:11 +0100

Failed: 3
Oldest: 2026-02-27 08:48:02 +0000
Newest: 2026-03-01 13:10:39 +0000

Pending: 12
Oldest: 2026-08-16 14:11:29 +0100
Newest: 2026-08-31 13:14:58 +0100


### Status report information ###

Generated at: 2026-08-16 14:55:19 +01:00
```
</details>

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.
